Utils/AppLogger.swift

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import XCGLogger
+
+public extension XCGLogger {
+    class func appInstance() -> XCGLogger {
+        struct statics {
+            static let instance: XCGLogger = {
+                var logger = XCGLogger.defaultInstance()
+                let cacheDir = NSSearchPathForDirectoriesInDomains(.CachesDirectory, .UserDomainMask, true).first as! String
+                let fileDestination = XCGFileLogDestination(owner: logger, writeToFile: "\(cacheDir)/app.log", identifier: XCGLogger.constants.baseFileLogDestinationIdentifier)
```

Comment:
`The value for writeToFile: can be a String or NSURL. If the file already exists, it will be cleared before we use it.`

So this is strictly better than we had, but we'll lose the last log every time the app opens. We need a log roller before this is useful.